.github/workflows/test.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         run: |
           version=0.0.$GITHUB_RUN_NUMBER
           echo "ThisBuild / version := \"$version\"" > version.sbt
-          echo "::set-output name=version::$version"
+          echo "version=$version" >> $GITHUB_OUTPUT
```

Comment:
Do we even need this in the 1st place? I don't see where it's used. I may be missing something obvious. It's been a long time since I wrote this!

Reply:
That is indeed a good question.